inception/inception-api-annotation-experimental/src/main/ts/client/visualization/AnnotationExperienceAPIVisualization.ts

```diff
@@ -0,0 +1,288 @@
+/*
+ * Licensed to the Technische Universität Darmstadt under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The Technische Universität Darmstadt
+ * licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {Annotation} from "../util/Annotation";
+import {AnnotationType} from "../util/AnnotationTypes";
+import {AnnotationExperienceAPI} from "../AnnotationExperienceAPI";
+
+export class AnnotationExperienceAPIVisualization {
```

Comment:
This looks like an example editor build on top of the API - it is not part of the API - IMHO "API" here in the class name is misleading to wrong